src/scenes/Gallery/components/Header/Header.tsx

```diff
@@ -28,9 +28,29 @@ function Header({ usernameOrWalletAddress }: Props) {
 
   return (
     <StyledHeader>
-      <Display>{usernameOrWalletAddress}</Display>
-      <Spacer height={36} />
+      <Subdisplay>{usernameOrWalletAddress}</Subdisplay>
+      <Spacer height={12} />
       <StyledUserDetails>
+        {isAuthenticated && (
+          <StyledRow>
+            <StyledRightContainer>
+              <Dropdown mainText="Edit Profile">
+                <TextButton
+                  text="Edit name & Bio"
+                  onClick={handleEditNameClick}
+                  underlineOnHover
+                />
+                <Spacer height={12} />
+                <TextButton
+                  text="Edit Gallery"
+                  onClick={handleEditGalleryClick}
+                  underlineOnHover
+                />
+              </Dropdown>
+            </StyledRightContainer>
+          </StyledRow>
```

Comment:
nit: this could be its own component `AuthenticatedOptions`

Reply:
good idea, I made an issue for it to keep this PR's scope limited: https://github.com/gallery-so/gallery/issues/234